### PR TITLE
Some clean-ups.

### PR DESCRIPTION
**Pull request by favonia**

### CODE REVIEW

**Comment by jonsterling** [comment]
looking good!

**Comment by jonsterling** [comment]
This is an exciting PR! nice work so far.

**Comment by favonia** [comment]
OMG I accidentally pushed stuff to the master branch (and rolled it back)...

File: src/redprl.mlb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ann
   "forceUsed"
   "warnUnused true"
+  "allowOrPats true"
```
Comment (favonia) [inline src/redprl.mlb]: I am departing from the standard Standard ML...

**Comment by favonia** [comment]
@jonsterling Hmm the MLton on travis does not support the annotation `allowOrPats true`. Not sure what to do.

**Comment by jonsterling** [comment]
@favonia It's too bad about the MLton thing... I guess we should avoid using successor ML features like that, unless we can figure out how to get a new version of MLton running on travis.

**Comment by jonsterling** [comment]
> OMG I accidentally pushed stuff to the master branch (and rolled it back)...

Haha, no worries! I think we've all done that by accident once or twice 😉

**Pull request merged**